core/chains/evm/config/toml/defaults/Arbitrum_Sepolia.toml

```
@@ -3,6 +3,7 @@ ChainType = 'arbitrum'
 NoNewHeadsThreshold = '0'
 OCR.ContractConfirmations = 1
 LogPollInterval = '1s'
+FinalityTagEnabled = true
```

nit: just to keep the ordering consistent across all of the configs, could you move `FinalityTagEnabled` under `ChainType` here?
```suggestion
FinalityTagEnabled = true
NoNewHeadsThreshold = '0'
OCR.ContractConfirmations = 1
LogPollInterval = '1s'
```